atlasdb-service-server/src/main/java/com/palantir/atlasdb/server/AtlasDbServiceServerConfiguration.java

```diff
@@ -25,7 +25,7 @@
 
     private final AtlasDbConfig atlasdb;
 
-    public AtlasDbServiceServerConfiguration(@JsonProperty(AtlasDbConfigs.ATLASDB_CONFIG_ROOT) AtlasDbConfig atlasdb) {
+    public AtlasDbServiceServerConfiguration(@JsonProperty(AtlasDbConfigs.ATLASDB_CONFIG_OBJECT_NAME) AtlasDbConfig atlasdb) {
```

Comment:
This is completely unrelated for now. This is what we name the configuration block which currently is called `atlas`, `atlasdb`, etc in products